sbin/start-master.sh

```diff
@@ -30,7 +44,7 @@ START_TACHYON=false
 while (( "$#" )); do
 case $1 in
     --with-tachyon)
-      if [ ! -e "$sbin"/../tachyon/bin/tachyon ]; then
+      if [ ! -e "${SPARK_HOME}"/../tachyon/bin/tachyon ]; then
```

Comment:
Last question: does the `..` need to go away? before it started with `$sbin/..` which I think means the parent of `sbin` which should be the project root?